.github/workflows/pr-build.yml

```diff
@@ -40,23 +40,4 @@ jobs:
       - name: Build with Gradle
         uses: hypertrace/github-actions/gradle@main
         with: 
-          args: build -x avroCompatibilityCheck dockerBuildImages
-
-  validate-helm-charts:
-    runs-on: ubuntu-20.04
-    container:
-      image: hypertrace/helm-gcs-packager:0.3.1
-      credentials:
-        username: ${{ secrets.DOCKERHUB_READ_USER }}
-        password: ${{ secrets.DOCKERHUB_READ_TOKEN }}
-      # Set fetch-depth: 0 to fetch commit history and tags for use in version calculation
-    steps:
-      - name: Check out code
-        uses: actions/checkout@v2.3.4
-        with:
-          ref: ${{github.event.pull_request.head.ref}}
-          repository: ${{github.event.pull_request.head.repo.full_name}}
-          fetch-depth: 0
-
-      - name: validate charts
-        run: ./.github/workflows/helm.sh validate
\ No newline at end of file
+          args: build -x avroCompatibilityCheck dockerBuildImages
```

Comment:
why is the compatibility check disabled?

Reply:
Since the avro was not published, will enable.